utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/schemaregistry/universal/ParsedSchemaSupport.scala

```diff
@@ -156,7 +158,20 @@ object JsonSchemaSupport extends ParsedSchemaSupport[OpenAPIJsonSchema] {
   override def formValueEncoder(schema: ParsedSchema, mode: ValidationMode): Any => AnyRef = {
     val encoder   = new ToJsonSchemaBasedEncoder(mode)
     val rawSchema = schema.cast().rawSchema()
-    (value: Any) => encoder.encodeOrError(value, rawSchema)
+    (value: Any) => {
+      // In ad-hoc test without schema we create object `{ "Value" = userInputInAdHoc }`, so if present we should just take the input
+      Try {
+        val temp = value.asInstanceOf[Map[String, Map[String, Any]]].head
+        if (temp._1.equals("Value")) {
```

Comment:
```scala
val temp = value.asInstanceOf[Map[String, Map[String, Any]]].head
```
can be replaces with
```scala
val (key, value) = value.asInstanceOf[Map[String, Map[String, Any]]].head
```

Reply:
I have switched it back a bit, because when I try to create val to store `temp._2` value it fails. It doesn't matter if I do it as you wrote or separately create `key` and `value`. 
It is strange because there is no problem with assigning `temp._1`